buildtools/dep/dep.go

```diff
@@ -2,17 +2,32 @@
 package dep
 
 import (
-	"errors"
+	"fmt"
 	"path"
-	"path/filepath"
+	"strings"
 
 	"github.com/fossas/fossa-cli/buildtools"
 	"github.com/fossas/fossa-cli/files"
 	"github.com/fossas/fossa-cli/pkg"
 )
 
-// ErrNoLockfile is returned if a dep manifest is found without an accompanying lockfile.
-var ErrNoLockfile = errors.New("found Gopkg.toml but not Gopkg.lock")
+// A Resolver contains both the Lockfile and Manifest information. Resolver implements Resolver.
+type Resolver struct {
+	Manifest Manifest
+	Lockfile Lockfile
+}
+
+// Manifest contains the contents of a dep toml file. Manifest implements Resolver.
+// Ignored is a result of toml unmarshalling. IgnoreMap is for easy search of ignored packages
```

Comment:
What is `IgnoreMap`?

Reply:
A leftover field that I chose to delete. Good catch, thank you.